Systems/c172p-damage.xml

```diff
@@ -288,20 +289,14 @@
             <test logic="AND" value="0.12">
                 /velocities/airspeed-kt GT damage/limits-vne-low
                 damage/roll-moment GT 4000.0
-                wing-damage/left-wing EQ 0.0
+                wing-damage/left-wing LT 0.12
```

Comment:
It would keep this at `EQ 0.0`. It means the left wing must be OK (not damaged or broken).